doc/api/documentation.md

```diff
@@ -32,6 +32,18 @@ The stability indices are as follows:
 > [semantic versioning][] rules. Non-backward compatible changes or removal may
 > occur in any future release. Use of the feature is not recommended in
 > production environments.
+>
+> Experimental features are subdivided into stages:
+>
+> * 1.0 - Early development. Experimental features at this stage are unfinished
+>   and subject to substantial change.
+> * 1.1 - Active development. Experimental features at this stage are nearing
+>   minimum viability.
```

Comment:
The text for 1.2 looks reasonable to me but 1.0 and 1.1 looks too similar because a feature that is "nearing minimum viability" is also "unfinished". I think we land experimental feature implementation PRs only when those are already in an MVP state which is not necessarily complete. What do you think about merging 1.0 and 1.1 into a single subdivision?

Reply:
So the purpose of 1.0 is to create space for iterative development; so a PR can land with a new API, say, but the API immediately crashes or doesn’t return anything etc. Rather than holding the bar at “this API needs to function as intended before it can merge,” which could mean either a large PR or development somewhere else like in a fork or other repo, the 1.0 status is to permit such unfinished work to land and iterate within the Node repo. It’s more a status for us, as in “don’t block this PR because it’s unfinished, it’s 1.0,” than it us for the public. For users the only message here is that it’s too early to open bug reports, because the feature is unfinished.

I can try to add some parts of this explanation to the text if you want. We’re not using this stage much at all today, but there have been times recently when it would have helped, such as https://github.com/nodejs/node/pull/45018.